lib/phoenix/code_reloader/proxy.ex

```diff
@@ -1,48 +1,120 @@
-# A tiny proxy that stores all output sent to the group leader
-# while forwarding all requests to it.
+# This proxy is spawned at startup and replaces (and forwards to)
+# the standard_error process.
 defmodule Phoenix.CodeReloader.Proxy do
   @moduledoc false
   use GenServer
 
-  def start() do
-    GenServer.start(__MODULE__, "")
+  def start_link() do
+    stderr = Process.whereis(:standard_error)
```

Comment:
It is worth noting that this process always runs... including in production. And I am not sure if that's ok.

Reply:
The Proxy is now only started when the Server needs it (2a36dc00eafd90dd6095fa06f10947a7d0e87f48)